back-end/poo-solid/day4/src/index.ts

```diff
@@ -0,0 +1,58 @@
+type Discipline = {
+  name: string;
+  grade: number;
+  letterGrade?: string;
+};
+
+type Student = {
+  name: string;
+  disciplines: Discipline[];
+};
+
+/* Converter */
+function percentageToLetterGrade({ name: studentName, disciplines }: Student):
```

Comment:
Essa função apresenta um erro de lint:

Arrow function has a complexity of 6. Maximum allowed is 5.